src/ModalHeader.js

```diff
@@ -4,24 +4,35 @@ import classnames from 'classnames';
 
 class ModalHeader extends React.Component {
 
+  renderCloseButton() {
+    if (this.props.closeButton === true) {
+      return (
+        <button
+          className='close'
+          aria-label={this.props['aria-label'] || 'Close'} //eslint-disable-line react/prop-types
+          onClick={this.props.onHide}
+          style={{ marginTop: -2 }}
```

Comment:
That isn't in the original Bootstrap code, what causes the discrepancy so we need to do this correction here?